fetcher/bin/fetch_gha_min_cost.py

```diff
@@ -0,0 +1,125 @@
+import os
+import json
+from datetime import datetime, timezone, timedelta
+import concurrent.futures
+from fetcher.services.github_service import GithubService
+import logging
+
+
+logging.basicConfig(
+    level=logging.INFO, format="%(asctime)s - %(name)s - %(levelname)s - %(message)s"
+)
+logger = logging.getLogger(__name__)
+
+
+def get_environment_variables() -> str:
+    github_app_moj_token = os.getenv("GH_APP_MOJ_TOKEN")
+    if not github_app_moj_token:
+        raise ValueError(
+            "The env variable GH_APP_MOJ_TOKEN is empty or missing")
+    github_app_ap_token = os.getenv("GH_APP_AS_TOKEN")
+    if not github_app_ap_token:
+        raise ValueError(
+            "The env variable GH_APP_AS_TOKEN is empty or missing")
+
+    return github_app_moj_token, github_app_ap_token
+
+
+def calculate_gha_cost_per_run(run_id, repo_object, github_service, os_multipliers, minute_cost_usd):
```

Comment:
"per_run" I think implies getting all the runs
"calculate_gha_run_minutes_and_cost"

type hints also useful here

Reply:
Fixed.